input.gms

```diff
@@ -80,4 +80,107 @@ $ondelim
+iNcon(NENSE) = 10; !! 11 different consumer size groups for non energy uses
+iNcon("BU") = 2;   !! ... except bunkers .
+
+
+* 11 vehicle mileage groups
+* 0.952 turned out to be a (constant) ratio between modal and average mileage through iterations in Excel
+
+iAnnSmallCons(runCy,"PC") = 0.5 * 0.952 * iTransChar(runCy,"KM_VEH","2010") * 1000 * 1E-6;
+iAnnConsLargCons(runCy,"PC") = 4 * 0.952 * iTransChar(runCy,"KM_VEH","2010") * 1000 * 1E-6;
+iAnnConsModCons(runCy,"PC") = 0.952 * iTransChar(runCy,"KM_VEH","2010") * 1000 * 1E-6;
+
+* modal value is assumed to be 2 tonnes/vehicle, min = 1/3*modal and max = 10*modal tkm.
+* 0.706 is the constant ratio of modal/average tkm through iterations in Excel
+iAnnSmallCons(runCy,"GU") = 0.5 * 0.706 * iTransChar(runCy,"KM_VEH_TRUCK","2010")* 1000 * 2 / 3  * 1E-6;
+iAnnConsLargCons(runCy,"GU") = 4 * 0.706 * iTransChar(runCy,"KM_VEH_TRUCK","2010") * 1000 * 2 * 10  * 1E-6;
+iAnnConsModCons(runCy,"GU") = 0.706 * iTransChar(runCy,"KM_VEH_TRUCK","2010") * 1000 * 2  * 1E-6;
+
+* Size will not play a role in buses, trains, ships and aircraft
+* Following values are given only for the sake of uniformity, but iDisFunConSize is not really calculated for non-road transport!
+
+iAnnSmallCons(runCy,"PA") = 40000 * 50 * 1E-6;
+iAnnConsLargCons(runCy,"PA") = 800000 * 300 * 1E-6;
+iAnnConsModCons(runCy,"PA") = 400000 * 100 * 1E-6;
+
+*iAnnSmallCons(runCy,"PB") = 20000 * 5 * 1E-6;
+*iAnnConsLargCons(runCy,"PB") = 200000 * 50 * 1E-6;
+*iAnnConsModCons(runCy,"PB") = 50000* 15 * 1E-6;
+
+iAnnSmallCons(runCy,"PT") = 50000 * 50 * 1E-6;
+iAnnConsLargCons(runCy,"PT") = 400000 * 500 * 1E-6;
+iAnnConsModCons(runCy,"PT") = 200000 * 150 * 1e-6;
+
+iAnnSmallCons(runCy,"GT") = 50000 * 20 * 1E-6;
+iAnnConsLargCons(runCy,"GT") = 400000 * 500 * 1E-6;
+iAnnConsModCons(runCy,"GT") = 200000 * 200 * 1e-6;
+
+*iAnnSmallCons(runCy,"PN") = 10000 * 50 * 1E-6;
+*iAnnConsLargCons(runCy,"PN") = 100000 * 500 * 1E-6;
+*iAnnConsModCons(runCy,"PN") = 50000 * 100 * 1e-6;
+
+iAnnSmallCons(runCy,"GN") = 10000 * 20 * 1E-6;
+iAnnConsLargCons(runCy,"GN") = 100000 * 1000 * 1E-6;
+iAnnConsModCons(runCy,"GN") = 50000 * 300 * 1e-6;
+
+iAnnSmallCons(runCy,INDSE) = 0.2  ;
+iAnnConsLargCons(runCy,INDSE) = 0.9 ;
+* assuming an average utilisation rate of 0.6 for iron & steel and 0.5 for other industry (see iterations in Excel):
+iAnnConsModCons(runCy,"IS") = 0.587;
+iAnnConsModCons(runCy,INDSE)$(not sameas(INDSE,"IS")) = 0.487;
+
+iAnnSmallCons(runCy,DOMSE) = iConsSizeDistHeat(runCy,"smallest")  ;
+iAnnConsLargCons(runCy,DOMSE) = iConsSizeDistHeat(runCy,"largest") ;
+iAnnConsModCons(runCy,DOMSE) = iConsSizeDistHeat(runCy,"modal");
+
+iAnnSmallCons(runCy,NENSE) = 0.2  ;
+iAnnConsLargCons(runCy,NENSE) = 0.9 ;
+* assuming an average utilisation rate of 0.5 for non-energy uses:
+iAnnConsModCons(runCy,NENSE) = 0.487 ;
+
+iAnnSmallCons(runCy,"BU") = 0.2 ;
+iAnnConsLargCons(runCy,"BU") = 1 ;
+iAnnConsModCons(runCy,"BU") = 0.5 ;
```

Comment:
this whole section (lines 92-158) needs to be reworked. to proceed, let's keep as it is for now and I will open an issue (assigning myself) to work on this